test/camera_test.py

```diff
@@ -1,5 +1,31 @@
 import unittest
+import pygame
+import pygame.camera
 
 
 class CameraModuleTest(unittest.TestCase):
     pass
+
+
+class TestPygameCamera(unittest.TestCase):
+    def setUp(self):
+        pygame.init()
+        pygame.camera.init()
+
+    def test_camera(self):
+        cameras = pygame.camera.list_cameras()
+        self.assertTrue(len(cameras) > 0, "No cameras found")
```

Comment:
If there's no cameras, it's probably better the test is skipped instead.

This way the test should work on more computers too.